bindings/rust/src/lib.rs

```diff
@@ -96,16 +96,70 @@ impl Drop for Instance {
     }
 }
 
+struct HostFunction {
+    module: CString,
+    name: CString,
+    inputs: Vec<sys::FizzyValueType>,
+    output: sys::FizzyValueType,
+    index: usize,
+}
+
+unsafe extern "C" fn host_callback(
+    host_context: *mut std::os::raw::c_void,
+    instance: *mut sys::FizzyInstance,
+    args: *const sys::FizzyValue,
+    context: *mut sys::FizzyExecutionContext,
+) -> sys::FizzyExecutionResult {
+    println!("host fuction called!");
+    //unimplemented!()
+    sys::FizzyExecutionResult {
+        trapped: false,
+        has_value: false,
+        value: sys::FizzyValue { i32: 0 },
+    }
+}
+
+fn create_function_import_list(
```

Comment:
Still need to figure out what is the best API defining host functions, but at least this semi-finished translation layer works.